sys/vm/vm_page.h

```diff
@@ -433,6 +433,21 @@ extern struct mtx_padalign pa_lock[];
  * PGA_SWAP_FREE is used to defer freeing swap space to the pageout daemon
  * when the context that dirties the page does not have the object write lock
  * held.
+ *
+ * PGA_CAPSTORE right now reflects the object's capability store permission.
```

Comment:
BTW, this comment might have been the source of my earlier confusion about PGA_CAPSTORE and the swap case with cap_store_on_alloc, etc.  I think the comment should talk more about the intended purpose and probably not claim that is identical to OBJ_HASCAP.